MiniGamesBox Inventory/src/main/java/plugily/projects/minigamesbox/inventory/paged/PagedFastInv.java

```diff
@@ -1,644 +1,161 @@
-  }
-
-  /**
-   * Add a handler to handle inventory close.
-   *
-   * @param closeHandler The handler to add
-   */
-  public void addCloseHandler(Consumer<InventoryCloseEvent> closeHandler) {
-    this.closeHandlers.add(closeHandler);
-  }
-
-  /**
-   * Add a handler to handle inventory click.
-   *
-   * @param clickHandler The handler to add.
-   */
-  public void addClickHandler(Consumer<InventoryClickEvent> clickHandler) {
-    this.clickHandlers.add(clickHandler);
-  }
-
-
-  /**
-   * Remove all handlers which handles inventory open.
-   */
-  public void removeOpenHandlers() {
-    this.openHandlers.clear();
-  }
-
-  /**
-   * Remove all handlers which handles inventory close.
-   */
-  public void removeCloseHandlers() {
-    this.closeHandlers.clear();
-  }
-
-  /**
-   * Remove all handlers which handles inventory click.
-   */
-  public void removeClickHandlers() {
-    this.clickHandlers.clear();
-  }
-
-  /**
-   * Open the inventory to a player.
-   *
-   * @param player The player to open the menu.
-   */
-  public void open(Player player) {
-    player.openInventory(getInventory(1));
-  }
-
-  /**
-   * Open the inventory to a player.
-   *
-   * @param player The player to open the menu.
-   */
-  public void open(Player player, int page) {
-    player.openInventory(getInventory(page));
-  }
-
-  /**
-   * Open the inventory to a player.
-   *
-   * @param humanEntity The humanEntity to open the menu.
-   * @param inventory   The inventory to open
-   */
-  public void open(HumanEntity humanEntity, Inventory inventory) {
-    humanEntity.openInventory(inventory);
-  }
-
-  /**
-   * Open the inventory to a player.
-   *
-   * @param humanEntity The humanEntity to open the menu.
-   */
-  public void open(HumanEntity humanEntity, int page) {
-    humanEntity.openInventory(getInventory(page));
-  }
-
-  /**
-   * Get borders of the inventory. If the inventory size is under 27, all slots are returned.
-   *
-   * @return inventory borders
-   */
-  public int[] getBorders(int page) {
-    int size = getInventory(page).getSize();
-    return IntStream.range(0, size).filter(i -> size < 27 || i < 9 || i % 9 == 0 || (i - 8) % 9 == 0 || i > size - 9).toArray();
-  }
-
-  /**
-   * Fill the page with items
-   *
-   * @param stack ItemStack to fill the inventory
-   * @param page  page to fill
-   */
-  public void fill(ItemStack stack, int page) {
-    while(getInventory(page).firstEmpty() != -1) {
-      setItem(getInventory(page), getInventory(page).firstEmpty(), stack, inventoryClickEvent -> inventoryClickEvent.setCancelled(true));
-    }
-  }
-
-  /**
-   * Fill all pages with items
-   *
-   * @param stack ItemStack to fill the inventory
-   */
-  public void fill(ItemStack stack) {
-    for(Inventory inventory : pagedInventories.values())
-      while(inventory.firstEmpty() != -1) {
-        setItem(inventory, inventory.firstEmpty(), stack, inventoryClickEvent -> inventoryClickEvent.setCancelled(true));
-      }
-  }
```

Comment:
Missing methods

Reply:
We use `ItemMap` to set items as a map. This `fill` will require the inventory as an argument, and I don't want that as it's annoying.
In such a case, just use `setDefaultItem`.